### ISSUE TITLE
No output for 001_generateBigWig.sh

### ISSUE THREAD

**Issue by nich-sugiarto** [shahcompbio/hmmcopy_refs#2]
I am currently building mappability files for the hg38 fasta file from Ensembl with bin size 51. However, even after 10 hours with 32GB of RAM, no outuput is being generated. Furthermore, there is no output in the log file, so there is no way for me to know if it is getting stuck at a step, or if it truly is running and needs more time. The only message I get is that a step may be deprecated soon. Is there a way to make the output verbose so I can confirm that it is running? 

I have attached the log message, as well as the code with my modifications (changing the location of some files for privacy).

Thank you in advance for your time.
**Code:**
```
#!/bin/bash
# Name of the job
#SBATCH --job-name=001_generateBigWig  # Name of the job

# Number of compute nodes
#SBATCH --nodes=1

# Number of cores
#SBATCH --cpus-per-task=1

#Number of memory
#SBATCH --mem-per-cpu=32GB

# Number of cores, in this case one
#SBATCH --ntasks-per-node=1

# Walltime (job duration)
#SBATCH --time=48:00:00

# Name of the output files to be created. If not specified the outputs will be joined
#SBATCH --output=${folder}/log/001_generateBigWig.%j.out
#SBATCH --error=${folder}/001_generateBigWig.%j.err


grch38=${folder3}/Homo_sapiens.GRCh38.dna.primary_assembly.fa

source ${condaFolder}/miniconda/etc/profile.d/conda.sh
conda activate hmmcopy

# add -b to the below commands if the references don't have a bowtie index
#/home/william1/bin/hmmcopy_utils/util/mappability/generateMap.pl -w 125 ${grch37} -o references/GRCh37-lite.ws_125.bw

# cut -f1,2 ${grch38}.fai > ${grch38}.sizes
# bowtie-build $grch38 $grch38
envs/hmmcopy_utils/util/mappability/generateMap.pl -w 51 ${grch38} -o references/GRCh38.ws_51.bw
```

**Log**
`Setting the index via positional argument will be deprecated in a future release. Please use -x option instead.
`

**Comment by marcjwilliams1** [comment]
Hi @nich-sugiarto, it's been a while since I used this so don't recall how long it should take. I would suggest you try a larger bin size (say a few Mb) and see if it works, I think that should be faster. Then at least you'll know if it's working or not.

**Comment by nich-sugiarto** [comment]
Hi,

Thanks for your response. I did try that, and it did work. 

That being said, I figured out the issue. Is it possible that `hmmcopy_utils` can't handle nonstandard chromosomes like the KI* and GL* group. I've removed those, and it worked fine. Those groups only took about a a bit of Mb (not even 100Mb) out of the whole 3Gb genome, so it's possible that was the issue. Especially since those chromosomes in my reference did not have the prefix chr in front.
